landing_page_backend/contacts/views.py

```diff
@@ -1,3 +1,22 @@
-from django.shortcuts import render
+from rest_framework.generics import CreateAPIView
+import csv
+from contacts.models import Contact
+from contacts.serializers import ContactSerializer
 
-# Create your views here.
+with open('data.csv', 'w') as file:
+    writer = csv.writer(file)
+    writer.writerow(
+        ('name', 'email', 'number', 'message')
+    )
+
+
+class ContactAPIView(CreateAPIView):
+    queryset = Contact.objects.all()
+    serializer_class = ContactSerializer
+    contacts = Contact.objects.all().values_list('name', 'email', 'number', 'message')
+    for contact in contacts:
+        with open('data.csv', 'a') as file:
```

Comment:
If we will run this script the second time, the file will contain data from the previous week.
We need a single file for every week, so I would rather make the name like "contacts-2022-09-01.csv"